cnd/src/http_api/route_factory.rs

```diff
@@ -20,7 +24,9 @@ pub fn new_action_link(id: &SwapId, action: &str) -> String {
     format!("{}/{}", swap_path(*id), action)
 }
 
-pub fn create<D: MetadataStore + StateStore + AliceSpawner + BobSpawner + Network + Clone>(
+pub fn create<
+    D: MetadataStore + StateStore + SpawnBob + Network + Clone + InitiateRequest + SendRequest,
+>(
     origin_auth: String,
     peer_id: PeerId,
     dependencies: D,
```

Comment:
:wrench: Maybe you should rename this to `connector`, since it is what you're passing in now. There may be other places where you can do the same thing?

Reply:
Done  .... except for @D4nte comment below :) (`con` vs `connector`).